app.py

```diff
@@ -4,6 +4,7 @@
 import json
 import requests
 from multiprocessing import Pool
+from slackclient import SlackClient
 
```

Comment:
I don't think we need `slackclient`. It can be done with a simple incoming webhook configured in Slack. We can just make POST request webhook to post messages in channels. 
Reference - https://github.com/kossiitkgp/kwoc/blob/master/app.py#L433-L445